metaworld_algorithms/config/rl.py

```diff
@@ -30,6 +30,7 @@ class OffPolicyTrainingConfig(TrainingConfig):
     warmstart_steps: int = int(4e3)
     buffer_size: int = int(1e6)
     batch_size: int = 1280
+    seq_len: int = 100
```

Comment:
This parameter is not needed in all OffPolicy algorithms and should not be here. You should subclass OffPolicyTrainingConfig to something like RecurrentOffPolicyTrainingConfig to add it